kyo-scheduler-pekko/jvm/src/test/scala/kyo/scheduler/KyoExecutorServiceConfiguratorTest.scala

```diff
@@ -0,0 +1,104 @@
+package kyo.scheduler
+
+import com.typesafe.config.ConfigFactory
+import java.util.concurrent.CountDownLatch
+import org.apache.pekko.actor.Actor
+import org.apache.pekko.actor.ActorSystem
+import org.apache.pekko.actor.Props
+import org.apache.pekko.testkit.TestKit
+import org.apache.pekko.testkit.TestProbe
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.NonImplicitAssertions
+import org.scalatest.freespec.AnyFreeSpecLike
+import scala.concurrent.Promise
+import scala.concurrent.duration.*
+
+class KyoExecutorServiceConfiguratorTest
```

Comment:
@He-Pin I think the last time I used Akka/Pekko was a decade ago :) Does it have custom locals/threadlocals that we need to check for propagation like in Finagle's integration?

Reply:
IIRC, NO, thread local is rarely used.